tmpl/.pre-commit-config.yaml

```diff
@@ -1,21 +1,67 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+-   repo: local
+    # Running hooks locally provides them access the dependencies
     hooks:
     -   id: check-ast
```

Comment:
IMHO running these checkers locally is overkill, as they are only used by pre-commit. It only makes sense for tools that are in `requirements.yml`, like `flake8` and `black`.

Reply:
I see your point and will happily change that back. I only see a small advantage adding `pre-commit-hooks` to the `dev-requirements.yml`, which is that the version control of all the hooks happens at exactly one place and not two.

Reply:
To be honest, I think for these hooks it probably doesn't really matter. So no strong opinion here. If you are both (@huppd and @ruestefa ) fine with the latest changes, we can merge.

Reply:
on second thought, if we make some hooks local, we should do it for all of them no? And we have these tools in `requirements.yml` if we add `pre-commit-hooks`. So the disadvantage is one additional dependency right? Anyways I'm fine with merging either way, just curious if I am missing a point here.

Reply:
If it's possible to run these hooks locally without having to expand each entry from typically one line to a multiline block in `.pre-commit-config.yml`, then either way is fine with me. However, if they need to be specified as verbosely as in the original PR, than that IMHO blows up the file too much for pretty much zero gain as the hooks are never run outside pre-commit and are generally simple enough that updating them is not that crucial.

Reply:
yep ok I'm still very indifferent but I see the point. I'll merge as is.

Reply:
But I'm not going to stand in the way if you decide on running them locally. :)